source/2020-07-24-the-ember-times-issue-157.md

```diff
@@ -70,14 +71,19 @@ We encourage you to check out Aswathprabhu's [explanation of reactivity](https:/
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## Blog post series of lessons learned working with Ember 📔
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Johan Ronsse (@Wolfr)](https://github.com/Wolfr) wrote a series of blog posts on working with Ember especially coming from a [designer’s point of view](https://mono.company/frontend/working-within-ember-designer-point-of-view/). The [blog](https://mono.company/frontend/getting-to-grips-with-ember/) goes through how the Angle Bracket usage and other features from Ember Octane makes invoking components just like writing HTML which helps designers to be able to focus on the things they care about.
```

Comment:
I think we can update the 2nd sentence to be shorter. What do you think about the following, which also removes the word "just (Alex lint)?

```suggestion
[Johan Ronsse (@Wolfr)](https://github.com/Wolfr) wrote a series of blog posts on working with Ember especially coming from a [designer’s point of view](https://mono.company/frontend/working-within-ember-designer-point-of-view/). The [blog](https://mono.company/frontend/getting-to-grips-with-ember/) explains how angle bracket and other Octane features make invoking components feel like writing HTML for designers.
```